mcux/mcux-sdk/devices/MIMXRT1181/all_lib_device_MIMXRT1181.cmake

```diff
@@ -96,6 +96,7 @@ list(APPEND CMAKE_MODULE_PATH
     ${CMAKE_CURRENT_LIST_DIR}/../../drivers/qtmr_1
     ${CMAKE_CURRENT_LIST_DIR}/../../drivers/rdc
     ${CMAKE_CURRENT_LIST_DIR}/../../drivers/rdc_sema42
+    ${CMAKE_CURRENT_LIST_DIR}/../../drivers/rtwdog
```

Comment:
Where are we pulling this patch from? Moreover, do we actually need it to enable support for the RTWDOG? If we don't, let's avoid including this to minimize the patches we have to track versus the MCUX SDK

Reply:
Updated. I asked NPI team. They're not used.